docs/91_version_history.rst

```diff
@@ -33,175 +33,111 @@ Release Date: 2025-03-13
 
     - New Features:
 
-       - The documentation has been updated to improve the discussion on tuning to encompass low through data ingestion.
+       - New installations now default to being secured with a username and password. Existing installation will not be impacted when upgraded.
```

Comment:
New installations will now default to be secured with a username and password. Upgrading existing installations will not affect their current settings.